package.json

```diff
@@ -24,12 +24,14 @@
     "test:tdd": "yarn clean && mocha --watch ./packages/**/test/**/**/*.spec.js --timeout 15000"
   },
   "devDependencies": {
+    "babel-eslint": "^10.0.3",
```

Comment:
It's required so that lint doesn't throw an error immediately within the editor. Yes I should move it to the package specifically.

Reply:
> It's required so that lint doesn't throw an error immediately within the editor. 

Not sure I understand what the editor has to do with our code though?  Does it actually cause  `eslint` to fail?

Maybe it would be better to provide a screenshot or the error message so I can see what the actual issue is, not sure this dependency is needed, at least not for the CLI since linting really shouldn't have any impact on our source code per se.

Reply:
yea it was highlighting the dynamic import, but I forgot I needed to make that string anyway so it doesn't matter actually, that entire dep is now not needed.